src/ProxyClient/MultiplexerClient.php

```diff
@@ -131,13 +165,19 @@ public function refresh($url, array $headers = [])
      * @param string $interface The FQN of the interface
      * @param string $method    The method to invoke
      * @param array  $arguments The arguments to be passed to the method
+     *
+     * @return mixed Return value of first method call
      */
     private function invoke($interface, $method, array $arguments)
     {
+        $result = null;
+
         foreach ($this->proxyClients as $proxyClient) {
             if (is_subclass_of($proxyClient, $interface)) {
-                call_user_func_array([$proxyClient, $method], $arguments);
+                $result = call_user_func_array([$proxyClient, $method], $arguments);
             }
         }
+
+        return $result;
```

Comment:
i would hope that usually the clients are the same implementation, so this solution seems fine to me. we should add some note in the documentation of the multipart client about this though.

however, i propose that for the getters we call the first client instead of using invoke. if we agree on that, the changes to the invoke method can be reverted.